src/client/pages/Checkout/CheckoutPayment/ContactInformation/ContactInformation.stories.tsx

```diff
@@ -11,12 +11,11 @@ import {
 } from 'utils/storybook/storyHelpers'
 import { localePathPattern } from 'l10n/localePathPattern'
 import { mockedQuoteCartId } from 'utils/testData/quoteDetailsDataMock'
-import { ContactInfoData } from '../../shared/types'
 import { ContactInformation } from './ContactInformation'
 
 type StoryProps = {
   localePath: LocaleLabel
-  data: ContactInfoData
+  data: any
```

Comment:
Isn't this also `FormikProps<QuoteInput>` like you have in the file below?

Reply:
yes it is but then formik requires another 10 values or something 😞 and this felt easier instead of giving empty strings. may have to rethink the way this story is created.